tv-shows-app/src/components/shared/ReviewForm/ReviewForm.tsx

```diff
@@ -65,22 +69,38 @@ export default function ReviewForm({ onAdd }: IReviewFormProps) {
     }
   }
 
+  function onEditingSubmit(data: IReviewFormInputs) {
+    onAdd(data);
+  }
+
+  useEffect(() => setValue("comment", initialComment || ""), []);
```

Comment:
I'm wondering why you need this. This implies that your rating can be changed from outside of the form. What is the use case that this handles?

Reply:
I'm reusing ReviewForm component for editing the review as well as submiting, so this allows to initially get the existing comment and edit it instead of having an empty input field when editing

Reply:
But do you need state for that? Did you try just passing it as a prop and it didn't work?

Reply:
I was already using register for changing comment and rating, InitialRating is a string prop and I set the comment to be initial comment in the beggining. Is there a better way to do this?